README.md

```diff
@@ -29,8 +29,8 @@ xortool
 
 Usage:
   xortool [-x] [-m MAX-LEN] [-f] [-t CHARSET] [FILE]
-  xortool [-x] [-l LEN] [-c CHAR | -b | -o] [-f] [-t CHARSET] [FILE]
-  xortool [-x] [-m MAX-LEN| -l LEN] [-c CHAR | -b | -o] [-f] [-t CHARSET] [FILE]
+  xortool [-x] [-l LEN] [-c CHAR | -b | -o] [-f] [-t CHARSET] [-kp PLAIN] [FILE]
```

Comment:
should be -p ? same in xortool.py

Reply:
@hellman Yes, fixed. Btw I wanted to add tests, but tests are failing on the master branch. I think you should config CI to check tests before merges after fixing the tests.

Reply:
I updated the installation, tests should work now ( v1.0.0 on PyPI  and master here)